config/routes.rb

```diff
@@ -4,14 +4,16 @@
     get 'users/:user_id/projects' => 'users#projects', as: :user_projects
 
     resources :projects do
-      resources :repositories, except: [:update, :index]
-      get '/repositories/:id/modules/:module_result_id' => 'repositories#show', as: :repository_module
-      post '/repositories/:id/state' => 'repositories#state', as: :repository_state
-      post '/repositories/:id/state_with_date' => 'repositories#state_with_date', as: :repository_state_with_date
-      put '/repositories/:id' => 'repositories#update', as: :repository_update
-      get '/repositories/:id/process' => 'repositories#process_repository', as: :repository_process
+      resources :repositories, only: [:new, :create]
     end
 
+    resources :repositories, except: [:update, :index]
```

Comment:
I think we are going to need an Index for repositories now that they can be created without a project. Or else they wont be accessible from anywhere in the website.